libs/ui/src/components/atoms/Radio/Radio.styles.ts

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import * as RadioGroupPrimitive from '@radix-ui/react-radio-group';
+import { Theme } from '../../../types/theming';
+import { font } from '../../../theme/global/font';
+
+export const Container = styled.div`
+  align-items: center;
+  display: flex;
+  padding-bottom: 14px;
```

Comment:
Use rem in spacing, heigh and width

Reply:
Switching from px to rem the size seems a bit off from the design sheet.

Reply:
Just a heads up, REM *should* be the same as px (1rem = 10px), provided that you are at 100% zoom.